capstone/src/test/java/com/google/univiz/api/resource/MapsResourceImplTest.java

```diff
@@ -0,0 +1,150 @@
+  private static final double NYU_LONGITUDE = -73.997264;
+  private static final int CARNEGIE_SIZE_DEGREE = 17;
+  private static final double NYU_ADMISSION_RATE = 0.1999;
+  private static final double NYU_AVG_SAT = 1419.0;
+  private static final int NYU_NUM_OF_UNDERGRADS = 26339;
+  private static final int NYU_AVG_COST = 69830;
+  private static final double NYU_RATIO_OF_MEN = 0.4253;
+  private static final double NYU_RATIO_OF_WOMEN = 0.5747;
+
+  private static final CollegeId STANFORD_COLLEGE_ID = CollegeId.create(243744);
+  private static final String STANDFORD_NAME = "Stanford University";
+  private static final String STANDFORD_CITY = "Stanford";
+  private static final boolean STANFORD_FLAG_MAIN_CAMPUS = true;
+  private static final double STANFORD_LATITUDE = 37.429434;
+  private static final double STANFORD_LONGITUDE = -122.167359;
+  private static final double STANFORD_ADMISSION_RATE = 0.0436;
+  private static final double STANFORD_AVG_SAT = 1497.0;
+  private static final int STANFORD_NUM_OF_UNDERGRADS = 7083;
+  private static final int STANFORD_AVG_COST = 69109;
+  private static final double STANFORD_RATIO_OF_MEN = 0.4991;
+  private static final double STANFORD_RATIO_OF_WOMEN = 0.5009;
+
+  private static final CollegeData NYU_COLLEGE_DATA =
+      CollegeData.builder()
+          .setId(NYU_COLLEGE_ID)
+          .setName(NYU_NAME)
+          .setCity(NYU_CITY)
+          .setIsMainCampus(NYU_FLAG_MAIN_CAMPUS)
+          .setLatitude(NYU_LATITUDE)
+          .setLongitude(NYU_LONGITUDE)
+          .setCarnegieSizeDegree(CarnegieSizeDegree.getDegree(CARNEGIE_SIZE_DEGREE))
+          .setAdmissionRate(NYU_ADMISSION_RATE)
+          .setAvgSat(NYU_AVG_SAT)
+          .setNumOfUndergrads(NYU_NUM_OF_UNDERGRADS)
+          .setAvgCost(NYU_AVG_COST)
+          .setRatioOfMen(NYU_RATIO_OF_MEN)
+          .setRatioOfWomen(NYU_RATIO_OF_WOMEN)
+          .build();
+
+  private static final CollegeData STANFORD_COLLEGE_DATA =
+      CollegeData.builder()
+          .setId(STANFORD_COLLEGE_ID)
+          .setName(STANDFORD_NAME)
+          .setCity(STANDFORD_CITY)
+          .setIsMainCampus(STANFORD_FLAG_MAIN_CAMPUS)
+          .setLatitude(STANFORD_LATITUDE)
+          .setLongitude(STANFORD_LONGITUDE)
+          .setCarnegieSizeDegree(CarnegieSizeDegree.getDegree(CARNEGIE_SIZE_DEGREE))
+          .setAdmissionRate(STANFORD_ADMISSION_RATE)
+          .setAvgSat(STANFORD_AVG_SAT)
+          .setNumOfUndergrads(STANFORD_NUM_OF_UNDERGRADS)
+          .setAvgCost(STANFORD_AVG_COST)
+          .setRatioOfMen(STANFORD_RATIO_OF_MEN)
+          .setRatioOfWomen(STANFORD_RATIO_OF_WOMEN)
+          .build();
+
+  @Before
+  public void guiceSetUp() {
+    Guice.createInjector(BoundFieldModule.of(this)).injectMembers(this);
+  }
+
+  @Test
+  public void testMapsResourceImpl() throws IOException {
+    List<CollegeId> ids = Lists.newArrayList(NYU_COLLEGE_ID);
+    assertThat(ids).hasSize(1);
```

Comment:
nit - no need to assert on the test data that you're creating (and throughout this test). typically you'll only need to assert on things that happen after the system under test is invoked.